blueprints/ember-fullcalendar/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('fullcalendar').then(() => {
-      return this.addBowerPackageToProject('fullcalendar-scheduler').then(() => {
-        return this.addAddonToProject('ember-cli-moment-shim', '0.6.2');
-      });
+    return this.addBowerPackageToProject('fullcalendar', '^2.7.3').then(() => {
```

Comment:
@DanChadwick is it actually necessary to specify the version here? I assumed it would just pull whatever was specified in `bower.json`. If that is the case, this may be redundant. Otherwise, educate me :)